Java/E-montazysta/src/main/java/com/emontazysta/configuration/DataSeeding.java

```diff
@@ -342,13 +341,13 @@ OrderStatus.TODO, new BigDecimal(4), 1, null, null, null,
                 "8:00 - 15:00", company2, location4, new ArrayList<>(), new ArrayList<>()));
 
         ToolType toolType1 = addToolTypeFromModel(new ToolType(null, "Test ToolType 1",
-                5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
+                5, 5, 5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
         ToolType toolType2 = addToolTypeFromModel(new ToolType(null, "Test ToolType 2",
-                5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
+                5, 5, 5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
         ToolType toolType3 = addToolTypeFromModel(new ToolType(null, "Test ToolType 3",
-                5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
+                5, 5, 5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
         ToolType toolType4 = addToolTypeFromModel(new ToolType(null, "Test ToolType 4",
-                5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
+                5, 5, 5, new ArrayList<>(), new ArrayList<>(), new ArrayList<>()));
```

Comment:
These values are derived attributes and shouldn't be passed while creating object, but calculated while we want information about object.